external/fv3fit/fv3fit/emulation/transforms/transforms.py

```diff
@@ -14,6 +14,36 @@ def backward(self, y: TensorDict) -> TensorDict:
         pass
 
 
+@dataclasses.dataclass
+class Difference(TensorTransform):
```

Comment:
Since this class doesn't depend on data, it felt silly to write a separate factory for it. This is why I added the `TransformFactory` APIs, `backward_names` and `build`.